kolibri/core/tasks/main.py

```diff
@@ -133,6 +134,9 @@ def __queue():
     Priority.HIGH: priority_queue,
 }
 
+# This storage instance should be used to access job_storage db
+job_storage = Storage(connection=connection)
```

Comment:
This may be the cause of the postgres errors you are seeing - make sure to follow the same lazy object pattern that the other objects defined in this module follow, rather than instantiating the `Storage` object in the module scope directly.

Reply:
Ohhh! Will fix this! :tada: 

But sir, why is this the case though? I mean any specific reason for following lazy object initialization...?

Reply:
I am not completely certain that this is the cause of your Postgresql failures - but the main reason for the lazy initialization is to allow objects from this module to be imported before any app initialization has happened, without fear of database connections being initialized.

Reply:
Got it!